templates/cf-apache.service.mustache

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=CFEngine Enterprise Webserver
+After=syslog.target
+After=cf-postgres.service
+ConditionPathExists={{{vars.sys.workdir}}}/httpd/bin/apachectl
+PartOf=cfengine3.service
+
+[Service]
+Type=forking
+ExecStart={{{vars.sys.workdir}}}/httpd/bin/apachectl start
+ExecStop={{{vars.sys.workdir}}}/httpd/bin/apachectl stop
+PIDFile={{{vars.sys.workdir}}}/httpd/logs/httpd.pid
+Restart=always
+RestartSec=10
+
+[Install]
+WantedBy=multi-user.target
+WantedBy=cfengine3.service
```

Comment:
I know I'm late to the party but just looking over the code and I notice we are dropping the units in /lib vs /etc.  I was always under the impression that custom systemd units should end up in /etc and not clobber the ones in /lib.

https://www.freedesktop.org/wiki/Software/systemd/FrequentlyAskedQuestions/

> Q: I want to change a service file, but rpm keeps overwriting it in /usr/lib/systemd/system all the time, how should I handle this?

> A: The recommended way is to copy the service file from /usr/lib/systemd/system to /etc/systemd/system and edit it there. The latter directory takes precedence over the former, and rpm will never overwrite it. If you want to use the distributed service file again you can simply delete (or rename) the service file in /etc/systemd/system again.